app/views/subscription/add.phtml

```diff
@@ -27,7 +27,7 @@
 		<div class="form-group">
 			<label class="group-name" for="url_rss"><?= _t('sub.feed.url') ?></label>
 			<div class="group-controls">
-				<input id="url_rss" name="url_rss" type="url" required="required" autocomplete="off" class="long"/>
+				<input id="url_rss" name="url_rss" type="url" required="required" autocomplete="off" class="long" autofocus="autofocus"/>
```

Comment:
Very minor, but while we are at it, the common convention is to put a space before ` />`

```suggestion
				<input id="url_rss" name="url_rss" type="url" required="required" autocomplete="off" class="long" autofocus="autofocus" />
```

Reply:
Only for compatibility with Netscape 4 or something along those lines. >_>

Reply:
Indeed I believe the convention came from some parsing-related aspect, to make sure it worked in all cases even in older products :-)

Reply:
I've changed it